static/templates/catalog_v2_entries.xml

```diff
@@ -49,6 +53,7 @@
     {{#url}}
     <link rel="http://opds-spec.org/acquisition/open-access" type="application/x-zim" href="{{{url}}}" length="{{{size}}}" />
     {{/url}}
+{{/dump_partial_entries}}
```

Comment:
I wonder if we could go a bit further and reuse the output of the template `catalog_v2_complete_entry.xml` here instead of almost copying the same template.

Reply:
Done.